go-controller/pkg/ovn/secondary_layer3_network_controller_test.go

```diff
@@ -149,11 +150,30 @@ var _ = Describe("OVN Multi-Homed pod operations", func() {
 				Expect(secondaryNetController.bnc.WatchNodes()).To(Succeed())
 				Expect(secondaryNetController.bnc.WatchPods()).To(Succeed())
 
-				// assert the OVN annotations on the pod are the expected ones
+				// check that after start networks annotations and nbdb will be updated
 				Eventually(func() string {
 					return getPodAnnotations(fakeOvn.fakeClient.KubeClient, podInfo.namespace, podInfo.podName)
 				}).WithTimeout(2 * time.Second).Should(MatchJSON(podInfo.getAnnotationsJson()))
 
+				defaultNetExpectations := getExpectedDataPodsAndSwitches([]testPod{podInfo}, []string{nodeName})
```

Comment:
should getExpectedDataPodsAndSwitches instead be called getDefaultNetExpectedPodsAndSwitches?